typhon/trees.py

```diff
@@ -8,7 +8,7 @@
 from collections.abc import Iterable
 
 import pandas as pd
-import numba
+# import numba
```

Comment:
Do you want to be consistent with the commented `number.jit` decorators at various places in the file?
I would rather remove `numba` completely 🤷‍♂️

Reply:
Yes, that was the idea, but ended up removing them. I needed they can always be retrieved from history if needed anyway.